content/post/logical-replication-webinar-2022.08.24/index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "PostgreSQL Logical Replication using KubeDB in Kubernetes"
+date: 2022-08-24
+weight: 26
+authors:
+ - Rakibul Hossain 
+tags:
+ - kubedb
+ - postgresql
+ - logical replication
+ - kubernetes
+ - publisher
+ - subscriber
+---
+
+## Summary
+On August 24, 2022, AppsCode held a webinar on **PostgreSQL Logical Replication using KubeDB in Kubernetes**. The key contents of the webinars are:
+1) PostgreSQL Logical Replication in KubeDB
+2) Features
+3) Demo
+4) Q & A Session
+
+## Description of the Webinar
+At first, we gave an overview of `PostgreSQL Logical Replication`.  To implement it in `KubeDB` we introduced two `CRD` `Publisher` & `Subscriber`, which will be controlled by `Logical Replication Operator`. `Publisher` will create and maintain `Publications` inside `KubeDB PostgreSQL Server`, and `Subscriber` will create and maintain `Subscription` inside `KubeDB PostgreSQL Server`.
+
+We described the features such as:
+ * Different Postgres Versions
+ * TableCreationPolicy
+ * Pre Conflict detection
+ * Double Opt-in
+
+Then we started the demonstration, at first we create a postgres server `publisher-db` in the `demo` namespace and a database `pub` with a table `table_1` inside the database. Then create a publisher `publisher-sample` in the same namespace, which will create and maintain a publication `my_pub` for table `table_1`. 
+
+After that we created another postgres server `subscriber-db` in the test namespace and a database `sub`. Then we create a subscriber `subscriber-one` in the same namespace, which will create and maintain a subscription `my_sub`  for the publication `my_pub` managed by the Publisher CRD. it also create the table `table_1` as we specified the `tableCreationPolicy: IfNotPresent` in the subscriber's `Spec`.
```

Comment:
`it` also create the table `table_1` as we specified the `tableCreationPolicy: IfNotPresent` in the subscriber's `Spec`.
Capitalize `it`.